CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # OverReact Changelog
 
+## [3.9.0](https://github.com/Workiva/over_react/compare/3.8.1...3.9.0)
+
+- [#620] Consume/update typing for forwardRef2/memo2 functions
```

Comment:
1. This should indicate that it's a bugfix it fixes bugs
2. There's another PR in here that needs release notes
```suggestion
- [#620] Fix issue where `uiForwardRef`/`memo` components were being passed JSified props (by consuming [react-dart's `forwardRef2`/`memo2`](https://github.com/cleandart/react-dart/pull/275))
- [#618] Add `makeMapStateToProps`/etc. arguments to `connect` to enable creation of closures specific to component instances
```